systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaMirrorMaker2Utils.java

```diff
@@ -62,8 +73,22 @@ public static void waitForKafkaMirrorMaker2ConnectorReadiness(String namespaceNa
         });
     }
 
-    public static boolean waitForKafkaMirrorMaker2StatusMessage(String namespaceName, String clusterName, String message) {
-        KafkaMirrorMaker2 kafkaMirrorMaker2 = KafkaMirrorMaker2Resource.kafkaMirrorMaker2Client().inNamespace(namespaceName).withName(clusterName).get();
-        return ResourceManager.waitForResourceStatusMessage(KafkaMirrorMaker2Resource.kafkaMirrorMaker2Client(), kafkaMirrorMaker2, message);
+    public static void waitForKafkaMirrorMaker2StatusMessage(String namespaceName, String clusterName, String message) {
+        LOGGER.info("Waiting for {}: {}/{} will contain desired status message: {}",
+            KafkaMirrorMaker2.RESOURCE_KIND, namespaceName, clusterName, message);
+
+        TestUtils.waitFor(String.format("%s: %s#%s will contain desired status message: %s", KafkaMirrorMaker2.RESOURCE_KIND, namespaceName, clusterName, message),
```

Comment:
Maybe you can use waitFor from test-frame? Or we are not planning to use it as we anyway cannot remove `TestUtils` ?

Reply:
it's copy pasted from different place, I will not rewrite everything in one PR as I mentioned previously.

Reply:
Ok, up to you